task/06-conditions-n-loops-tasks.js

```diff
@@ -30,7 +30,13 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+    return (num % 3 === 0 && num % 5 === 0)
+                ? 'FizzBuzz'
+                : (num % 3 === 0)
+                    ? 'Fizz'
+                    : (num % 5 === 0)
+                        ? 'Buzz'
+                        : num;
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators ? :